documentation/docker.md

```diff
@@ -241,17 +241,13 @@ To use `docker` commands, either directly, or through the abstraction of the Mak
 - You will require write access to GitHub packages [https://github.com/DFE-Digital/teaching-vacancies/](https://github.com/DFE-Digital/teaching-vacancies/pkgs/container/teaching-vacancies) repository. Ask in #digital-tools-support should you require it.
 - Log in to Log in to Github Packages (with `docker login ghcr.io -u USERNAME` - use PAT token as passoword)
 
-#### The `twd-tv-ci` user:
```

Comment:
This section is much longer, please delete it completely

Reply:
As discussed, create a small section explaining the authentication from the workflow